cschwitalla/Ligandomics_Analysis/Ligandomics_Analysis.R

```diff
@@ -52,8 +60,8 @@ for(i in rewritten_hlatypes){
 #Benign data Immunology -> from Marissa ==================================
 #more specific
 #less hits 
-Benigniome_I = read.csv("/Users/cschwitalla/Documents/Ligandomics_analysis/newBenignmorespecific/Benign_class1.csv", header = FALSE, sep = ",", col.names = c("Peptide", "Acc", "V3", "Tissue", "V5"))
-Benigniome_II = read.csv("/Users/cschwitalla/Documents/Ligandomics_analysis/newBenignmorespecific/Benign_class2.csv", header = FALSE, sep = ",", col.names = c("Peptide", "Acc", "V3", "Tissue", "V5","V6"))
+Benigniome_I = read.csv("/Users/cschwitalla/Documents/Immunopeptidomics/newBenignmorespecific/Benign_class1.csv", header = FALSE, sep = ",", col.names = c("Peptide", "Acc", "V3", "Tissue", "V5"))
```

Comment:
Is this `Ligandomics_analysis` the same as `Immunopeptidomics` ?
If this is the case then you can also use a relative path. You already set a new working directory on the top